config/repos.yml

```diff
@@ -62,6 +62,68 @@ master:
   manageiq-v2v:
   ui-components:
     skip_tag: true
+morphy:
+  amazon_ssa_support:
+  container-amazon-smartstate:
+    skip_tag: true
+  container-httpd:
+    has_real_releases: true
+  container-memcached:
+    has_real_releases: true
+  container-postgresql:
+    has_real_releases: true
+  container-ruby:
+    has_real_releases: true
+  dbus_api_service:
+    has_real_releases: true
+  httpd_configmap_generator:
+    has_real_releases: true
+  manageiq:
+    has_rake_release: true
+  manageiq-api:
+  manageiq-appliance:
+  manageiq-appliance-build:
+  manageiq-appliance_console:
+    has_real_releases: true
+  manageiq-automation_engine:
+  manageiq-consumption:
+  manageiq-content:
+    has_rake_release: true
+  manageiq-decorators:
+  manageiq-documentation:
+  manageiq-gems-pending:
+  manageiq-graphql:
+  manageiq-pods:
+  manageiq-providers-amazon:
+  manageiq-providers-ansible_tower:
+  manageiq-providers-autosde:
+  manageiq-providers-azure:
+  manageiq-providers-azure_stack:
+  manageiq-providers-foreman:
+  manageiq-providers-google:
+  manageiq-providers-ibm_cloud:
+  manageiq-providers-ibm_terraform:
+  manageiq-providers-kubernetes:
+  manageiq-providers-kubevirt:
+  manageiq-providers-lenovo:
+  manageiq-providers-nsxt:
+  manageiq-providers-nuage:
+  manageiq-providers-openshift:
+  manageiq-providers-openstack:
+  manageiq-providers-ovirt:
+  manageiq-providers-redfish:
+  manageiq-providers-scvmm:
+  manageiq-providers-vmware:
```

Comment:
@agrare I intentionally didn't include ibm_power_vc and oracle_cloud, so you'll have to update this when those branches are ready

Reply:
:+1: perfect